calabash/Classes/FranklyServer/Operations/LPOperation.m

```diff
@@ -69,6 +70,8 @@ + (id) operationFromDictionary:(NSDictionary *) dictionary {
     operation = [[LPQueryAllOperation alloc] initWithOperation:dictionary];
   } else if ([opName isEqualToString:@"setText"]) {
     operation = [[LPSetTextOperation alloc] initWithOperation:dictionary];
+  } else if ([opName isEqualToString:@"clearText"]) {
+    operation = [[LPClearTextOperation alloc] initWithOperation:dictionary];
```

Comment:
I think this be a route that operates on the first responder.  It should not act on views that do not have keyboard visible keyboard attached (no quite the right way to say that).  If there is no first responder the route should respond with an error.